src/WebAPI/Controllers/WeatherForecastController.cs

```diff
@@ -6,6 +8,20 @@ namespace WebAPI.Controllers
     [Route("[controller]")]
     public class WeatherForecastController : ControllerBase
     {
-        
+        private readonly IMediator _mediator;
+        public WeatherForecastController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+        [HttpGet("currentWeather/{city}")]
+        public async Task<double?> GetCurrentWeather([FromRoute] string city)
+        {
+            //await _mediator.Send(new CurrentWeatherCommand());            
+            var result = await _mediator.Send(new CurrentWeatherCommand
```

Comment:
no need for `result` variable, just return.

Reply:
Done